### DISCUSSION TITLE
How to compute the capacity loss according to the Loss of active material

### DISCUSSION THREAD

**Discussion by Mrzhang-hub** [pybamm-team/PyBaMM#1945] [Q&A]
I simultaneously considered three side reactions in my model including Loss of Lithium-ion(LLI), SEI growth and Loss of active material(LAM). The pybamm package provide the method to acquire the capacity loss while I could not find the approach how to calculate the capacity loss according to the LAM in the pybamm package.

**Comment by valentinsulzer** [comment]
How do you define capacity loss due to LAM?

**Reply by Mrzhang-hub** [reply]
> How do you define capacity loss due to LAM?

![微信图片_20220221091604](https://user-images.githubusercontent.com/59716473/154874040-475424ec-21c8-4015-a444-afdf9c060436.png)

**Reply by Mrzhang-hub** [reply]
> How do you define capacity loss due to LAM?

The essay define the capacity loss caused by LAM through this equation. I wanna know why it is defined like this and if pybamm  will implement this in the future.

**Reply by valentinsulzer** [reply]
I don't know how they derived that specific equation, but you can find similar derivations in the appendices of this paper https://iopscience.iop.org/article/10.1149/1945-7111/ac3e48/pdf. My current understanding is that it is not possible to explictly define dQam/dt as a function of deps_am/dt, due to the nonlinearity in the way capacity is calculated. However, we could add the contribution from dCn/dt to dnLi/dt, as in eq [40] of the linked paper

**Reply by Mrzhang-hub** [reply]
Thank u for your reply. And I have one more question to consult.  Is that normal that capacity loss calculated from Lithium plating  far outweigh that calculated from SEI growth, with the former  10-5 and the  latter 10-9. 
![Li_plating](https://user-images.githubusercontent.com/59716473/155723722-07d4bda5-f052-4bd6-82f5-9634bcad596d.png)
